include/psa/crypto_struct.h

```diff
@@ -231,6 +232,8 @@ typedef struct psa_tls12_prf_key_derivation_s
     size_t MBEDTLS_PRIVATE(seed_length);
     uint8_t *MBEDTLS_PRIVATE(label);
     size_t MBEDTLS_PRIVATE(label_length);
+    uint8_t *MBEDTLS_PRIVATE(other_secret);
```

Comment:
I think this new pair of fields should be guarded by
```
#if defined(MBEDTLS_PSA_BUILTIN_ALG_TLS12_PSK_TO_MS)
```
since it's specific to that algorithm (contrary to the other fields, which are common with `MBEDTLS_PSA_BUILTIN_ALG_TLS12_PRF`).